Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -38,47 +41,54 @@ public EditorHandData(uint handId)
             public bool IsFingerDownPending;
             public bool FingerStateChanged;
             public float FingerStateUpdateTimer;
-			public float FingerDownStartTime;
+            public float FingerDownStartTime;
+            public bool ManipulationInProgress;
+            public bool HoldInProgress;
+            public Vector3 CumulativeDelta;
         }
 
         private ManualHandControl manualHandControl;
 
         /// <summary>
-        /// Dispatched each frame that a hand is moving
+        /// Dispatched each frame that a hand is moving.
         /// </summary>
         public event Action<IInputSource, uint> HandMoved;
 
         /// <summary>
         /// Delay before a finger pressed is considered.
         /// This mitigates fake finger taps that can sometimes be detected while the hand is moving.
         /// </summary>
-        private const float FingerPressDelay = 0.07f;
+        private const float fingerPressDelay = 0.07f;
```

Comment:
My take on it is that it should be CamelCase (with uppercase fist letter). That's what it is internally in the team that wrote most of these addons in the first place.
That being said, in case of doubt, I'd say we should do the minimal change and not change how variables are named when we don't need to, unless it's a clear bad naming convention violation. This prevents unnecessary conflicts with other changes that might have pending by other contributors. So in this case, they'd stay with their original CamelCase naming.

Otherwise this change looks good to me